apps/builder/app/builder/features/sidebar-left/help-popover.tsx

```diff
@@ -0,0 +1,110 @@
+import {
+  Button,
+  Flex,
+  Popover,
+  PopoverContent,
+  PopoverTrigger,
+  styled,
+  theme,
+} from "@webstudio-is/design-system";
+import {
+  DiscordIcon,
+  GithubIcon,
+  TwitterIcon,
+  WebstudioLogoFlatIcon,
+  YoutubeIcon,
+} from "@webstudio-is/icons";
+import { type ComponentProps } from "react";
+
+const StyledLink = styled("a", {
+  textDecoration: "none",
+  color: "white",
+  display: "inline-flex",
+  cursor: "default",
+});
+
+const PopoverItemButton = ({
+  href,
+  ...buttonProps
+}: ComponentProps<typeof Button> & { href?: string }) => (
+  <StyledLink href={href} target="_blank" referrerPolicy="no-referrer">
+    <Button
+      css={{
+        width: "100%",
+      }}
+      {...buttonProps}
+    />
+  </StyledLink>
+);
+
+export const HelpPopover = ({
```

Comment:
this interface shouldn't forward all popover properties, its a higher level abstraction, specifically for help, so only pass props that are actually needed